components/Contributors/OscrPill.tsx

```diff
@@ -1,17 +1,35 @@
 import Pill from "components/atoms/Pill/pill";
 import Tooltip from "components/atoms/Tooltip/tooltip";
+import { supabase } from "lib/utils/supabase";
 
-export const OscrPill = ({ rating, hideRating }: { rating: number | undefined; hideRating: boolean }) => {
+const DEFAULT_SIGN_IN = supabase.auth.signInWithOAuth.bind(supabase.auth);
+
+export const OscrPill = ({
+  rating,
+  hideRating,
+  signIn = DEFAULT_SIGN_IN,
```

Comment:
I opted for a default sign in method mainly so I can pass a prop in in a Storybook story, otherwise I would have had to mock the sign in.

I also don't pass in the `signIn` prop everywhere there is an `<OscrPill />` component because it's honestly a lot of ceremony/prop drilling with no value really as we always want a CTA to log in on this component if the user is logged out/is a guest.